### PR TITLE
DE-4444 | Wikianalytics design feedback

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/DE-4444

And https://wikia-inc.atlassian.net/browse/DE-4440 - extension enabled site-wide.

## Do not merge yet!

### CODE REVIEW

File: extensions/wikia/Hydralytics/Hydralytics.i18n.php [hunk 1]
```diff
@@ -30,10 +30,10 @@
 	'edits_per_day' => '# of Edits per Day',
 	'logged_in_out' => 'Logged In vs. Out Edits',
 	'edits' => 'Edits',
-	'top_search_terms' => 'Top Search Terms',
+	'top_search_terms' => 'Internal Top Search Terms',
```
Comment (MikolajBalcerek) [inline extensions/wikia/Hydralytics/Hydralytics.i18n.php:26]: Remember to push this to crowdin :)

File: extensions/wikia/Hydralytics/Hydralytics.i18n.php [hunk 2]
```diff
@@ -30,10 +30,10 @@
...
 	'search_rank' => 'Search rank',
 	'search_term' => 'Search term',
-	'analytics_report_generated' => 'Report Generated $1',
+	'analytics_report_generated' => 'Report generated $1',
```
Comment (macbre) [inline extensions/wikia/Hydralytics/Hydralytics.i18n.php:29]:
```
macbre@dev-macbre-18:~/app/extensions/wikia/Hydralytics$ crowdin upload
[INFO]            Converting MediaWiki i18n file to JSON
[INFO]            Uploading source file for project ext-hydralytics to Crowdin.
[INFO]            Successfully uploaded /tmp/crowdin.ext-hydralytics.macbre/build/Hydralytics.json to Crowdin.
```

**Review by MikolajBalcerek — APPROVED** [review]

File: extensions/wikia/Hydralytics/Hydralytics.i18n.php [hunk 3]
```diff
@@ -666,7 +558,6 @@
 	'number_of_pageviews' => '頁面訪問次數',
 	'right-analytics' => '訪問維基數據',
 	'desktop_vs_mobile' => '電腦桌面 vs 手機',
-	'error_analytics_usage_master' => '數據使用頁面只存在於主維基中。',
```
Comment (macbre) [inline extensions/wikia/Hydralytics/Hydralytics.i18n.php:678]: I've uploaded these changes to crowdin.

**Comment by MikolajBalcerek** [comment]
Merged changes from DE-4451 and uploaded translation changes from this ticket as well

**Pull request merged**